client/cli/src/lib.rs

```diff
@@ -869,18 +398,18 @@ fn input_keystore_password() -> Result<String, String> {
 }
 
 /// Use in memory keystore config when it is not required at all.
-fn fill_config_keystore_in_memory<C, G, E>(config: &mut sc_service::Configuration<C, G, E>)
+fn fill_config_keystore_in_memory<G, E>(config: &mut sc_service::Configuration<G, E>)
 	-> Result<(), String>
 {
 	match &mut config.keystore {
-		cfg @ KeystoreConfig::None => { *cfg = KeystoreConfig::InMemory; Ok(()) },
-		_ => Err("Keystore config specified when it should not be!".into()),
+		 cfg @ KeystoreConfig::None => { *cfg = KeystoreConfig::InMemory; Ok(()) },
```

Comment:
```suggestion
		cfg @ KeystoreConfig::None => { *cfg = KeystoreConfig::InMemory; Ok(()) },
```

Reply:
@cecton ???

Reply:
Sorry didn't see it for some reason. It appears in the PR files tab but not in the summary tab.